### PR TITLE
XW-3961 | use extracted jwplayer-fandom

### PR DESCRIPTION
**Pull request by latata**
https://wikia-inc.atlassian.net/browse/XW-3961
@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/ArticleVideo/scripts/featured-video.autoplay.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
 define('wikia.articleVideo.featuredVideo.autoplay', ['wikia.cookies', 'wikia.geo', 'wikia.instantGlobals'], function (cookies, geo, instantGlobals) {
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/featured-video.autoplay.js]: MAJOR Remove the unused function parameter "geo". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
MAJOR Remove the unused function parameter "instantGlobals". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

File: extensions/wikia/ArticleVideo/ArticleVideo.hooks.php [hunk 2]
```diff
@@ -67,6 +67,8 @@ private static function addOoyalaAssets(\OutputPage $out, $featuredVideoData, $r
 	}
 
 	private static function addJWPlayerAssets() {
+		// fixme that causes extra request
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/ArticleVideo.hooks.php]: MAJOR Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js [hunk 3]
```diff
@@ -1,96 +1,93 @@
...
 	}
 
-	var videoId = videoDetails.mediaId,
-		inNextVideoAutoplayCountries = featuredVideoAutoplay.inNextVideoAutoplayCountries,
+	var inNextVideoAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoNextVideoAutoplayCountries),
 		//Fallback to the generic playlist when no recommended videos playlist is set for the wiki
 		recommendedPlaylist = videoDetails.recommendedVideoPlaylist || 'Y2RWCKuS',
-		willAutoplay = featuredVideoAutoplay.willAutoplay,
-		pausedOnRelated = false;
+		autoplayCookieName = 'featuredVideoAutoplay',
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js]: MAJOR Remove the declaration of the unused 'autoplayCookieName' variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
MAJOR Remove this useless assignment to local variable "autoplayCookieName" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 5 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js [hunk 4]
```diff
@@ -1,96 +1,93 @@
...
 
-	function setupPlayer(bidParams) {
-		logger.info('jwplayer setupPlayer');
-		wikiaJWSettings();
-		playerInstance.setup({
-			advertising: {
-				autoplayadsmuted: willAutoplay,
-				client: 'googima'
+	function setupPlayer() {
+		wikiaJWPlayer('featured-video__player', {
```
Comment (xkxd) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js]: `window.wikiaJwPlayer`?

**Review by xkxd — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/ArticleVideo/ArticleVideo.hooks.php (hunk 2 above)
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/ArticleVideo.hooks.php]: MAJOR Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)

[project-id]: wikia:app

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[project-id]: wikia:app
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AssetsManager/config.php [hunk 5]
```diff
@@ -2694,7 +2694,6 @@
 	'assets' => [
 		'//extensions/wikia/ArticleVideo/styles/jwplayer.scss',
```
Comment (hakubo) [inline extensions/wikia/AssetsManager/config.php:2697]: import all scss files in scss

**Review by hakubo — APPROVED** [review]

File: extensions/wikia/ArticleVideo/templates/ArticleVideo_jwfeatured.php [hunk 6]
```diff
@@ -12,7 +12,6 @@ class="featured-video__title"><?= htmlspecialchars( $videoDetails['title'] ) ?><
 			<?= DesignSystemHelper::renderSvg( 'wds-icons-cross-tiny', 'wds-icon wds-icon-tiny featured-video__close' ) ?>
 		</div>
 		<?= $app->renderPartial( 'ArticleVideo', 'feedback' ) ?>
-		<script src="//content.jwplatform.com/libraries/VXc5h4Tf.js"></script>
```
Comment (hakubo) [inline extensions/wikia/ArticleVideo/templates/ArticleVideo_jwfeatured.php:15]: yay!

**Pull request merged**